### PR TITLE
Change WITHOUT_OPENSHIFT flag to true in egeria 

### PR DESCRIPTION
**Pull request by rohithdv**
This PR partially fixes PR #422 . 
Signed-off-by: rohithdv <rovallam@in.ibm.com>

### CODE REVIEW

**Review by shlomitk1 — APPROVED** [review]

File: third_party/egeria/deploy.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set -e
 set -x
 
-: ${WITHOUT_OPENSHIFT=false}
+: ${WITHOUT_OPENSHIFT=true}
```
Comment (ronenkat) [inline third_party/egeria/deploy.sh:6]: I believe it should be :=true and not =true

Comment (rohithdv) [reply third_party/egeria/deploy.sh]: Fixed. Thanks.

Comment (roee88) [reply third_party/egeria/deploy.sh]: If that's true then why not change the other variable too?

Comment (rohithdv) [reply third_party/egeria/deploy.sh]: fixed for the DONOT_DELETE_PERSISTENT_STORAGE flag also. I think = or := is equivalent. But it is better to have a consistent notation.

**Pull request merged**